src/lib/othent/othent.ts

```diff
@@ -949,17 +965,29 @@ export class Othent implements Omit<ArConnect, "connect"> {
 
     // This function returns a new signed transaction. It doesn't mutate/sign the original one:
     const transactionToSign = await arweave.createTransaction({
```

Comment:
I added all the missing `Transaction` properties that we were not using before in here.